packages/common/src/cis2/util.ts

```diff
@@ -431,6 +449,38 @@ function serializeCIS2OperatorOfQuery(query: CIS2.OperatorOfQuery): Buffer {
     return Buffer.concat([owner, address]);
 }
 
+export function tokenAddressFromBase58(str: Base58String): CIS2.TokenAddress {
```

Comment:
I also expect a function TokenAddress -> String.

Maybe I missed it somewhere else?

Reply:
I didn't catch that you wanted both, but I'll add it